### PR TITLE
Pds mer mi test

### PR DESCRIPTION
**Pull request by gsn9**
Added a test to cover the PDS3 MerMI template, as well as some small fixes to the template. 

## Description

Fixed a few small issues with the MerMI template and created a test to test the MerMI Template. Also uploaded a cropping notebook for Mer images.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [x] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Comment by gsn9** [comment]
I will work on case two for the Mer MI tests tomorrow.

**Comment by gsn9** [comment]
Case two is added, and ready for review

File: isis/tests/FunctionalTestsIsisImportMerMI.cpp [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+
+#include "gmock/gmock.h"
+
+using namespace Isis;
+using json = nlohmann::json;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/isisimport.xml").expanded();
+
+
+TEST_F(TempTestingFiles, FunctionalTestIsisImportMerMICaseOne){
```
Comment (scsides) [inline isis/tests/FunctionalTestsIsisImportMerMI.cpp:25]: Is there a better name for these. If it is just that way from history that ok, but if during the conversion we find out how "CaseOne" is different from "CaseTwo" we should name them something with meaning.

File: isis/tests/data/isisimport/1M189529263EFF64KCP2977M2F1_cropped.IMG [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+TARGET_TYPE                    = PLANET
+APPLICATION_PROCESS_ID         = 29
+APPLICATION_PROCESS_NAME       = MI
+APPLICATION_PROCESS_SUBTYPE_ID = 1
+EARTH_RECEIVED_START_TIME      = "2006-01-06T22:13:45.946Z"
+EARTH_RECEIVED_STOP_TIME       = "2006-01-06T22:14:04.177Z"
+EXPECTED_PACKETS               = N/A
+PACKET_MAP_MASK                = N/A
+RECEIVED_PACKETS               = N/A
+SPICE_FILE_NAME = /oss/merb/ops/ops/surface/strategic/ref/naif/misc/chronos/chronos.mer
```
Comment (scsides) [inline isis/tests/data/isisimport/1M189529263EFF64KCP2977M2F1_cropped.IMG]: We can't have full paths in files anymore. If the beginning slash is changed to a space, not deleted, then all should be good and we meet our requirements

File: isis/tests/data/isisimport/2M189392700EFFAL00P2977M2F1_cropped.IMG [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+TARGET_TYPE                    = PLANET
+APPLICATION_PROCESS_ID         = 29
+APPLICATION_PROCESS_NAME       = MI
+APPLICATION_PROCESS_SUBTYPE_ID = 1
+EARTH_RECEIVED_START_TIME      = "2006-01-01T19:04:43.874Z"
+EARTH_RECEIVED_STOP_TIME       = "2006-01-01T19:05:11.255Z"
+EXPECTED_PACKETS               = N/A
+PACKET_MAP_MASK                = N/A
+RECEIVED_PACKETS               = N/A
+SPICE_FILE_NAME = /oss/mera/ops/ops/surface/strategic/ref/naif/misc/chronos/chronos.mer
```
Comment (scsides) [inline isis/tests/data/isisimport/2M189392700EFFAL00P2977M2F1_cropped.IMG]: same full path problem here

**Review by scsides — CHANGES_REQUESTED** [review]

File: isis/tests/FunctionalTestsIsisImportMerMI.cpp (hunk 1 above)
Comment (gsn9) [reply isis/tests/FunctionalTestsIsisImportMerMI.cpp:25]: I don't know of a better name for these tests, maybe @acpaquette has a better idea of naming these tests.

**Review by scsides — APPROVED** [review]

**Pull request merged**